packages/PowerGrid/src/webgpu/scatterplot.ts

```diff
@@ -22,6 +22,18 @@ class WebGPUCache {
   markerSizesLength = -1;
   markerDefaultSize = -1;
   sizeColumnName = '';
+  colorBuffer: GPUBuffer | null = null;
+  colorColumnName = '';
+  colorLength = -1;
+  colorAxisType: keyof typeof DG.AxisType = DG.AxisType.logarithmic;
```

Comment:
do I get this right that you initialize it to log because you expect a missmatch when linear comes by default?